tests/test_other.py

```diff
@@ -3595,6 +3595,10 @@ def test_dashM_respect_dashO(self):
     assert len(with_dash_o) == 0
     assert len(without_dash_o) != 0
 
+  def test_null_device_as_input(self):
+    out = Popen([PYTHON, EMXX, '-E', '-dM', '-xc', '/dev/null'], stdout=PIPE).communicate()[0]
+    self.assertContained('#define __EMSCRIPTEN__ 1', out) # Verify output contains one of the native predefined macros
```

Comment:
this comment still doesn't make it clear enough to me why checking for that output is enough for this test. When that output is present, what does it prove worked?